website/identifiers/tasks.py

```diff
@@ -8,7 +8,7 @@
 @queued_task
 @celery_app.task(ignore_results=True)
 def update_doi_metadata_on_change(target_guid):
-    sentry.log_message(f'Updating DOI for [{target_guid}]')
+    sentry.log_message('Updating DOI for guid', extra_data={'guid': target_guid})
```

Comment:
As long as you're here, can you try replacing `sentry.log_message` with `sentry.sentry.captureMessage` so that this doesn't show up as an error? (That's the right thing, correct, @mfraezz?)

Reply:
WibbleBibble does it [like this](https://docs.sentry.io/platforms/python/enriching-events/scopes/#using-with-scope), setting the `level` to `info` for "expected" errors (e.g. a 404 for a file that isn't there)

Reply:
We override something in sentry so that doesn't work 🤦

Reply:
looks like `log_message` is just a wrapper for `sentry.captureMessage`?

Reply:
A [wrapper](https://github.com/CenterForOpenScience/osf.io/blob/develop/framework/sentry/__init__.py#L38-L51) which doesn't pass kwargs like `level`, but yes. Could either pass `**kwargs` or append `level=logging.ERROR` to the signature and pass only that.

Reply:
ah, gotcha -- added `level=logging.ERROR` param and set it here to `level=logging.INFO`